common-arcana.lic

```diff
@@ -83,6 +83,13 @@ module DRCA
     fix_standing
   end
 
+  def use_glyph(name)
+    ability_data = get_data('spells').paladin_abilities[name]
+    result = DRC.bput("#{ability_data[name]['use_command']}", ability_data[name]['activated_message'], 'only to look foolish', 'this glyph is already in effect here')
+    return false if result =~ /only to look foolish/
```

Comment:
Can you `return result !~ /only to look foolish/` instead of lines 89 and 90? If so, the `return` is implicit and could be removed as well (so the line would just be `result =~ /only to look foolish/`)

Reply:
Right, changed it.